clients/venice-admin-tool/src/main/java/com/linkedin/venice/AdminTool.java

```diff
@@ -604,6 +609,28 @@ private static void showSchemas(CommandLine cmd) {
     printObject(valueSchemas);
   }
 
+  private static void executeDataRecovery(CommandLine cmd) {
+    String cluster = getRequiredArgument(cmd, Arg.CLUSTER);
```

Comment:
I have some contradictory requirements for this point in my mind:

1. If we only consider triggering the command, in particularly the `repush` command, it doesn't have a requirement on cluster info, meaning that we don't need to care about cluster at all.
2. On the other hand, if we want to make data recovery idempotent later (as you pointed out in the below comments), maybe we want to achieve that by updating stores Znode with some data recovery related progress info, and stores Znode are grouped by clusters in ZK. Would restricting stores to live in the same cluster make the updating part a bit easier, (only one controller api call to update, instead of figuring out which cluster, then update)?
3. This implementation also allow user to give `CLUSTER` only without providing store names, and in that case, it would pick all user stores in that cluster.

If point 2 (multiple controller api calls) is not a concern for us later, I think we might just make the `CLUSTER` optional (or a dummy cluster name as you suggested) or introduce a new `CLUSTER_TO_RECOVERY` for point 3.

Reply:
This is a usability improvement as we discussed in the meeting.
If we could improve the usability of the recovery tool by adding some acceptable complexity to the implementation, I think it worths the effort.

Otherwise, in the future when someone is trying to use the tool, they would raise the similar concerns.